### ISSUE TITLE
Postinstall script 'snyk protect' fails

### ISSUE THREAD

**Issue by RIAstar** [robohydra/robohydra#21]
The post-install script `snyk-protect` fails when installing version `0.6.3` of RoboHydra on my system (Windows 7).
This is the relevant part of the log:
```
verbose stack Error: robohydra@0.6.3 postinstall: `npm run snyk-protect`
verbose stack Exit status 1
verbose stack     at EventEmitter.<anonymous> (~\node-5.4.1\node_modules\npm\lib\utils\lifecycle.js:232:16)
verbose stack     at emitTwo (events.js:87:13)
verbose stack     at EventEmitter.emit (events.js:172:7)
verbose stack     at ChildProcess.<anonymous> (~\node-5.4.1\node_modules\npm\lib\utils\spawn.js:24:14)
verbose stack     at emitTwo (events.js:87:13)
verbose stack     at ChildProcess.emit (events.js:172:7)
verbose stack     at maybeClose (internal/child_process.js:821:16)
verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:211:5)
verbose pkgid robohydra@0.6.3
verbose cwd ~\.dev\centric\shell
error Windows_NT 6.1.7601
error argv "~\\node-5.4.1\\node.exe" "~\\node-5.4.1\\node_modules\\npm\\bin\\npm-cli.js" "i" "robohydra@0.6.3" "--save-dev"
error node v5.4.1
error npm  v3.5.4
error code ELIFECYCLE
error robohydra@0.6.3 postinstall: `npm run snyk-protect`
error Exit status 1
error Failed at the robohydra@0.6.3 postinstall script 'npm run snyk-protect'.
error Make sure you have the latest version of node.js and npm installed.
error If you do, this is most likely a problem with the robohydra package,
error not with npm itself.
error Tell the author that this fails on your system:
error     npm run snyk-protect
error You can get information on how to open an issue for this project with:
error     npm bugs robohydra
error Or if that isn't available, you can get their info via:
error     npm owner ls robohydra
error There is likely additional logging output above.
verbose exit [ 1, true ]
```
All the environment information you require should be in that log.
I would think the node and npm versions are sufficiently up-to-date.

Installation of version `0.6.2` works without a problem.

**Comment by emanchado** [comment]
Bah. I have just released 0.6.4 without it. I don't understand why it doesn't work (seems to work on my machine) but I don't really care for having snyk run on post-install, so I've removed it.

Let me know if you have problems with 0.6.4.

**Comment by RIAstar** [comment]
I have no clue what `snyk` is really ;)
Thanks for your quick reply. `0.6.4` is installing and working fine.
